Subject: Upgrading CFI query table for the new Intel flash type
From: Juergen Beisert <j.beisert@pengutronix.de>

The new flash used on Phytec's pcm030 CPU card uses a 1.5 revision cfi query
table.
FIXME: Hot fix. Side effects are to be checked.

Signed-off-by: Juergen Beisert <j.beisert@pengutronix.de>

---
 drivers/mtd/chips/cfi_cmdset_0001.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: drivers/mtd/chips/cfi_cmdset_0001.c
===================================================================
--- drivers/mtd/chips/cfi_cmdset_0001.c.orig
+++ drivers/mtd/chips/cfi_cmdset_0001.c
@@ -276,7 +276,7 @@ read_pri_intelext(struct map_info *map, 
 		return NULL;
 
 	if (extp->MajorVersion != '1' ||
-	    (extp->MinorVersion < '0' || extp->MinorVersion > '4')) {
+	    (extp->MinorVersion < '0' || extp->MinorVersion > '5')) {
 		printk(KERN_ERR "  Unknown Intel/Sharp Extended Query "
 		       "version %c.%c.\n",  extp->MajorVersion,
 		       extp->MinorVersion);
